locales/es/translation.json

```diff
@@ -230,5 +230,11 @@
   "PC Link Mode": "PC Link Mode",
   "afterLink": "",
   "Make sure the meter is switched off and plug in with micro-USB cable": "Asegúrese de que el medidor esté apagado y conéctelo con un cable micro-USB",
-  "Please correct the date/time on the linked pump.": "Corrija la fecha / hora en la bomba vinculada."
+  "Please correct the date/time on the linked pump.": "Corrija la fecha / hora en la bomba vinculada.",
+  "e-instructions": "e-instructions",
```

Comment:
Tämän oikeaa käännöstä varten tarvittaisiin ehkä espanjalainen käännös siitä ISO-standardista?

Toinen vaihtoehto olisi selvittää, löytyykö termi siitä säädöksestä, joka löytyy kaikilla kielillä osoitteesta https://eur-lex.europa.eu/legal-content/FI/TXT/?uri=CELEX:32021R2226. Pikaisella vilkaisulla ei näyttänyt olevan.

Reply:
En löytänyt espanjalaista käännöstä tuosta, joten muutin "e-instruction" tekstin "consult instructions for use", tuolla lailla se taitaa olla englanniksi, jonka sitten käänsin muille kielille.

Reply:
Osaako @HannuHyttinen auttaa tässä? Voisi kysellä muualta regulaatioasiantuntijoilta kuinka noudattavat näitä merkintästandardeja. Hankkivatko erikseen käännöksen joka kielelle?

Reply:
Mikaelin laittamassa linkissä asetus on kaikilla EU:n virallisilla kielillä. Virallinen asetuksen mukainen termi löytyy artiklan 2 alakohdasta 1. Itse suosisin mieluummin käyttäjän helposti ymmärtämää lausetta 15223 standardin kapulakielen sijaan (esim. standardin englanninkielinen termi "eIFU indicator" vaatisi minusta selityksen, jos sitä käyttäisi). 

Asetuksen "virallisia" käännöksiä:
FI = sähköinen käyttöohje
EN = instructions for use in electronic form
ES = instrucciones de uso en formato electrónico